core/src/main/java/jenkins/model/AssetManager.java

```diff
@@ -83,41 +83,26 @@ public void doDynamic(StaplerRequest req, StaplerResponse rsp) throws IOExceptio
             return null;
         }
 
-        try {
-            if (path.contains("..")) // crude avoidance of directory traversal attack
-                throw new IllegalArgumentException(path);
+        if (path.contains("..")) // crude avoidance of directory traversal attack
+            throw new IllegalArgumentException(path);
 
-            String name;
-            if (path.charAt(0) == '/') {
-                name = "assets" + path;
-            } else {
-                name = "assets/" + path;
-            }
+        String name;
+        if (path.charAt(0) == '/') {
+            name = "assets" + path;
+        } else {
+            name = "assets/" + path;
+        }
```

Comment:
revert

Reply:
why? the try catch was removed so indentation changed